internal/stack/serverless.go

```diff
@@ -29,9 +30,11 @@ const (
 
 	ParamServerlessLocalStackVersion = "serverless_local_stack_version"
 
-	configRegion          = "stack.serverless.region"
-	configProjectType     = "stack.serverless.type"
-	configElasticCloudURL = "stack.elastic_cloud.host"
+	configRegion              = "stack.serverless.region"
+	configProjectType         = "stack.serverless.type"
+	configElasticCloudURL     = "stack.elastic_cloud.host"
+	configProjectFleetPeriod  = "stack.serverless.fleet.period"
+	configProjectFleetTimeout = "stack.serverless.fleet.timeout"
```

Comment:
Maybe we should be more specific about the use of these settings.
```suggestion
	configProjectFleetPeriod  = "stack.serverless.fleet.wait_period"
	configProjectFleetTimeout = "stack.serverless.fleet.wait_timeout"
```